evalai/challenges.py

```diff
@@ -67,10 +69,50 @@ def list_future_challenges():
     get_future_challenge_list()
 
 
+@click.group(invoke_without_command=True)
+@click.pass_context
+@click.option('-c', '--challenge-id', type=int,
+              help="Challenge ID for viewing its phases.")
+@click.option('-p', '--phase-id', type=int,
+              help="Phase ID for showing the phase details")
+def phases(ctx, challenge_id, phase_id):
+    """
+    Displays phase and phase related details.
+    Invoked by running `evalai challenges phases`
+    """
+    if ctx.invoked_subcommand != "list":
+        if challenge_id is None or phase_id is None:
+            echo("Please pass in both parameters.")
```

Comment:
Please make this error message clear to end user. You can also give the command which is to be used actually.